docs/docs/concepts.mdx

```diff
@@ -143,7 +144,22 @@ We have some standardized parameters when constructing ChatModels:
 
 ChatModels also accept other parameters that are specific to that integration.
 
+#### Tool Calling
```

Comment:
should we consolidate this with the `#functiontool-calling` section further down?

Reply:
yes good catch, missed that section!